manifests/prometheus/alerts.d/diego_cell_mem_few_candidates.yml

```diff
@@ -0,0 +1,26 @@
+# Source: bosh-exporter
+---
+- type: replace
+  path: /instance_groups/name=prometheus2/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: DiegoCellMemoryFewCandidates
+    rules:
+      - alert: DiegoCellMemoryFewCandidates4G
+        expr: (count(firehose_value_metric_rep_capacity_remaining_memory{bosh_job_name="diego-cell"} > 4096) or vector(0)) < 3
+        for: 10m
+        labels:
+          severity: critical
+        annotations:
+          summary: Few candidate cells for a 4GiB app
+          description: Only {{ $value | printf "%.0f" }} cell(s) are advertising enough free memory to run a 4GiB app instance. Some tenant app(s) may be scaling out of control or we may have depleted cell capacity.
```

Comment:
Could we tweak this to make it more explicit 

1. what the user impact could be
2. what an operator needs to do

Reply:
+1 to these

Reply:
If you want to add that, you can add that. I'm not adding that because I don't have definite or succinct answers for either of those. And if we did have definite answers for either of those, why wouldn't we just script the action?